pkg/config/config.go

```diff
@@ -223,8 +223,8 @@ func NewConfig(confString string, c *cli.Context) (*Config, error) {
 				{Mime: webrtc.MimeTypeOpus},
 				{Mime: webrtc.MimeTypeVP8},
 				{Mime: webrtc.MimeTypeH264},
-				// {Mime: webrtc.MimeTypeAV1},
-				// {Mime: webrtc.MimeTypeVP9},
+				{Mime: webrtc.MimeTypeAV1},
```

Comment:
It this enabling both codecs by default?

Reply:
@boks1971 I push 3 commits for the solve since something is missed in my first solve comment (disable av1 codec, remove the comments you mentioned below), sorry for those might hard to review :)